browser-entry.js

```diff
@@ -129,7 +129,7 @@ mocha.setup = function(opts) {
     opts = {ui: opts};
   }
   for (var opt in opts) {
-    if (opts.hasOwnProperty(opt)) {
+    if (Object.prototype.hasOwnProperty.call(opts, opt)) {
```

Comment:
What's the reasoning behind this change?

Reply:
The [`no-prototype-builtins`](https://eslint.org/docs/rules/no-prototype-builtins) rule is enabled by [`eslint-config-standard`](https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json#L110), and this started erroring when I updated to the latest. The purpose of the rule is to ensure that a shadowed property isn't being called on the object.